server/admin/member.js

```diff
@@ -58,4 +64,48 @@ module.exports = function createJoinHandler(ringpop) {
             }));
         });
     };
+}
+
+function createLeaveHandler(ringpop) {
+    return function handleLeave(arg1, arg2, hostInfo, callback) {
+        if (typeof callback !== 'function') {
+            callback = function noop() {};
+        }
+
+        if (!ringpop.membership.localMember) {
+            process.nextTick(function() {
+                callback(errors.InvalidLocalMemberError());
+            });
+            return;
+        }
+
+        if (ringpop.membership.localMember.status === 'leave') {
```

Comment:
In other places we're codifying strings to objects, a la Status.Start. Should this also be codified?